connections/src/app/features-michael/profile-loader/profile-loader.component.ts

```diff
@@ -1,10 +1,55 @@
 import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Links } from 'src/app/core/models/links';
+import { LinksService } from 'src/app/core/services/links.service';
+import { PiiService } from 'src/app/core/services/pii.service';
 
 @Component({
   selector: 'app-profile-loader',
   templateUrl: './profile-loader.component.html',
   styleUrls: ['./profile-loader.component.css']
 })
 export class ProfileLoaderComponent {
+  inputForm: FormGroup;
+  bio: string;
+  links: Links[] = [];
 
+  constructor(private fb: FormBuilder, private piiService: PiiService, private linkService: LinksService){
+    this.inputForm = this.fb.group({
+      userId: ['', Validators.required]
+    });
+    this.bio = '';
+  }
+
+  onSubmit(event: Event): void {
+    //keep validation check for form
+    // if (this.piiform.valid) {
+    event.preventDefault();
+
+    console.log("User Id Value", this.inputForm.value);
+    const id = this.inputForm.value.userId;
+    this.piiService.getUserPii(id).subscribe({
+      next: response => {
+        this.bio = response.biography;
+        console.log('Bio get successfully:', this.bio);
+        this.inputForm.reset();
```

Comment:
Looks good to me but i can see cors error.
Need to check this.

Reply:
as long as your coming from http://localhost:4200/
you can't be seeing CORS.